pkg/operator/sync.go

```diff
@@ -56,53 +57,80 @@ func (optr *Operator) syncAll(config *OperatorConfig) error {
 		return nil
 	}
 
+	errors := []error{}
 	// Sync webhook configuration
 	if err := optr.syncWebhookConfiguration(); err != nil {
+		errors = append(errors, fmt.Errorf("Error syncing machine API webhook configurations: %w", err))
```

Comment:
The problem with this is we're increasing the likelihood that webhooks aren't running on first install (maybe other scenarios?) prior to the machine-api controllers being deployed.  This is going to cause bad things to happen down the road.

Reply:
yeh webhooks should fail early